mux/src/tab.rs

```diff
@@ -1417,6 +1417,8 @@ impl TabInner {
         if let Some(panel_idx) = self.get_pane_direction(direction, false) {
             self.set_active_idx(panel_idx);
         }
+        let mux = Mux::get();
+        mux.notify(MuxNotification::WindowInvalidated(self.id));
```

Comment:
`WindowInvalidated` is for a window, but here we are a tab.
This likely just so happens to work because the ids for both of these start at 0.

What you may want to try instead is the `PaneOutput` notification, and emit it for both the prior active and the newly active pane ids.

Reply:
I'm having trouble getting `PaneOutput` to work correctly. It does fix the issue of only working for the initial window, however it results in this error each time `wezterm cli activate-pane-direction` is called:
 ```
 22:47:57.991  ERROR  wezterm_mux_server_impl::local > encoding PDU to client: writing pdu data buffer: An existing connection was forcibly closed by the remote host. (os error 10054)
 ```
 I'm really not sure what's causing it, nor how to fix it as it seems to happen regardless of how I call notify with `PaneOutput`.

Reply:
Can you share the latest version of your code?

Reply:
I've pushed my changes, although my latest attempt was to try to send `PaneOutput` to every pane.
Some other things I've tried include: 
- using `panel_idx` (not sure if 'pane' and 'panel' are different in this context)
- using `self.iter_panes().iter().find(|pane| pane.is_active)`.
- hard coding a value (just to check)

They each result in the same error.

Reply:
Can you try this instead?:

```diff
diff --git a/mux/src/tab.rs b/mux/src/tab.rs
index 89a256878..4b5922395 100644
--- a/mux/src/tab.rs
+++ b/mux/src/tab.rs
@@ -1416,6 +1416,11 @@ impl TabInner {
         }
         if let Some(panel_idx) = self.get_pane_direction(direction, false) {
             self.set_active_idx(panel_idx);
+
+            let mux = Mux::get();
+            if let Some(window_id) = mux.window_containing_tab(self.id) {
+                mux.notify(MuxNotification::WindowInvalidated(window_id));
+            }
         }
     }
```

Reply:
FWIW, I think `panel_idx` may be a typo. I don't remember writing that!

Reply:
That works!